bindings/julia/Displaz.jl

```diff
@@ -79,14 +117,24 @@ interpret_color(s::String) = length(s) == 1 ? interpret_color(s[1]) : error("Unk
 interpret_color(c::Char) = color_names[c]
 
 
+const shape_names = @compat Dict('.' => [0],
+                                 's' => [1],
+                                 'o' => [2],
+                                 'x' => [3],
+                                 '+' => [4],
+								 '-' =>	[5])
+
+interpret_shape(markershape) = markershape
+interpret_shape(c::Char) = shape_names[c]
+
 # True if plots are to be plotted over the previous data, false to clear before
 # plotting new data sets.
 _hold = false
 
-
 # Basic 3D plotting function for points
 function plot(position; color=[1 1 1], markersize=[0.1], markershape=[0])
```

Comment:
Sure, I'm happy to rename some things.  As I said somewhere else, this whole API needs a rethink...  The underlying issue is that there's not a well established convention for plotting APIs in julia.  Or at least there wasn't when I started this.

Since then there's been a lot of development.  One of the most interesting packages from an API point of view is an attempt to unify plotting here:

https://github.com/tbreloff/Plots.jl

I really like the convention from Plots.jl of doing away with `hold()` and just using `plot()` and `plot!()`.

As far as naming goes, `plot3()` could make sense I suppose.